vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureReportGenerator.java

```diff
@@ -111,16 +115,30 @@ private void generateReport()
         LOGGER.info("Allure report is successfully generated at: {}", reportDirectory.getAbsolutePath());
     }
 
-    private void writeExecutorInfo() throws IOException
+    private void writeCategoriesInfo() throws IOException
     {
-        ExecutorInfo executorInfo = propertyMapper.readValue("allure.executor.", ExecutorInfo.class);
+        List<AllureCategory> categories = List.of(
+                new AllureCategory("Test defects", List.of(Status.BROKEN)),
+                new AllureCategory("Product defects", List.of(Status.FAILED)),
+                new AllureCategory("Known issue", List.of(Status.UNKNOWN)));
+        createJsonFileInResultsDirectory(objectMapper, "categories.json", categories);
+    }
 
-        try (BufferedWriter writer = Files.newBufferedWriter(resultsDirectory.toPath().resolve("executor.json")))
+    private void createJsonFileInResultsDirectory(ObjectMapper objectMapper, String fileName, Object content)
```

Comment:
as you decided to make `ObjectMapper` a field of class, there is no reason to pass it as input parameter to the method of the same class